index.src.html

```diff
@@ -845,16 +845,13 @@ <h3 id="determine-requests-referrer" dfn export>
         <dd>
           <ol>
             <li>
-              If |environment| is not null:
-              <ol>
-                <li>
-                  If |environment|'s <a for="environment settings object">HTTPS state</a> is "`modern`" <em>and</em>
-                  <var>request</var>'s
-                  <a href="https://fetch.spec.whatwg.org/#concept-request-current-url">current
-                  URL</a> is not a <a>potentially trustworthy
-                  URL</a>, then return <code>no referrer</code>.
-                </li>
-              </ol>
+              If |referrerURL| is <code>no referrer</code>, then return <code>no referrer</code>.
```

Comment:
I don't think instances of this are necessary, since the spec indicates that Fetch will not call into this algorithm if the referrer string is `no referrer`. Do you agree?